### ISSUE TITLE
Line Temperature Definition

### ISSUE THREAD

**Issue by berkdemir** [strusoft/femdesign-api#676]
Hi,
Probably since it is not used a lot, temperature definition has some problems in my opinion.

**Grasshopper Definition**
First of all TopBotLocationValue.Construct is not clear on its purpose. The output is same as the Point.
![image](https://user-images.githubusercontent.com/40423004/213786705-6e6dda50-c84f-4517-a4c0-b2a5582c8359.png)
And you cannot cast this to the line temperature definition, because it requests 2 points. (Although we don't know what do these points mean.)
![image](https://user-images.githubusercontent.com/40423004/213786863-a5c14c41-88d6-43e7-a598-14e2242481fc.png)
The text at the top is not correct, in my opinion. Because, according to this note we have to be able to put 1 point, but as you can at the bottom if statement, it doesn't allow anything different than 2.
https://github.com/strusoft/femdesign-api/blob/d21b699a30eadef16a43edde24586bddafe94e25/FemDesign.Core/Loads/LineTemperatureLoad.cs#L38-L58
If we accept and put two points (again, without knowing what they do), it does something weird. There are many beams to give uniform temperature of 100, but it just puts a weird sign to one of them. However, I fed all the curves of the beams.
![image](https://user-images.githubusercontent.com/40423004/213787380-2e8e507d-9008-4ac3-92bd-b00cc1d53325.png)
To summarize the end goal: I have more than 1 beams that I want to assign uniform temperature as shown below. I couldn't figure out how with the Grasshopper.
![image](https://user-images.githubusercontent.com/40423004/213787691-114ce450-3c37-43c0-9564-9f256401529e.png)

**Comment by Marco-Pellegrino** [comment]
Hi @berkdemir 

I do understand your concern and I'll create an example on our documentation page (see note on the other issue regarding the `Slab.VariableThicknessObject`)

In order to achieve what you have asked, I would suggest to apply the following Grasshopper Snippets.

![image](https://user-images.githubusercontent.com/53620289/214001293-8fb28949-fb60-478e-9eb0-8af7c874759e.png)

Using the datatree makes everything easier to understand what happen behind the "component" as every curve is matching the values in the branches.

Most of our users issue is due to the concept of "item" or "list". As you can see, `TopBotLocValue `require to be a `list input` while the rest is `item`. It takes some time to understand how those are running and my suggestion is to "graft" everything in order to have the same number of branch for every element :)

![image](https://user-images.githubusercontent.com/53620289/214002680-11576da9-33d3-4059-bdf3-8d5920169859.png)


Regarding the input `Point` of `TopBotLocValue`.

The `Point` input is to specify a subdomain of a line. I do think that it should be removed as we are already specifying the line geometry in the main component.

![image](https://user-images.githubusercontent.com/53620289/214001987-9bbcd1cc-719f-4537-b1e3-215fe8704101.png)

I hope everything's is clear.
